intro_flutter_gpu/step_09/lib/main.dart

```diff
@@ -198,4 +148,42 @@ class TrianglePainter extends CustomPainter {
 
   @override
   bool shouldRepaint(covariant CustomPainter oldDelegate) => true;
+
+  static const List<List<double>> vertices = [
+    // Vertex format: [x, y, z, u, v, w]
+
+    // Front face of cube
+    [-0.5, -0.5, 0.5, 0.0, 0.0, 1.0],
+    [0.5, -0.5, 0.5, 1.0, 0.0, 1.0],
+    [0.5, 0.5, 0.5, 1.0, 1.0, 1.0],
+    [-0.5, 0.5, 0.5, 0.0, 1.0, 1.0],
+
+    // back face of cube
+    [-0.5, -0.5, -0.5, 0.0, 0.0, 0.0],
+    [0.5, -0.5, -0.5, 1.0, 0.0, 0.0],
+    [0.5, 0.5, -0.5, 1.0, 1.0, 0.0],
+    [-0.5, 0.5, -0.5, 0.0, 1.0, 0.0],
+  ];
+
+  // Define indices for triangles (counter-clockwise winding)
+  static const indices = [
+    // Front face
+    0, 2, 1, 0, 3, 2,
+    // Back face
+    4, 5, 6, 4, 6, 7,
+    // Left face
+    0, 4, 7, 0, 7, 3,
+    // Right face
+    1, 2, 6, 1, 6, 5,
+    // Bottom face
+    0, 1, 5, 0, 5, 4,
+    // Top face
+    3, 7, 6, 3, 6, 2,
+  ];
+
+  /// Flattened vertex data ready for insertion in graphics buffer.
+  /// The vertex format is [x,y,z,u,v,w].
```

Comment:
Consider using code font here as `[...]` is generally used for links or element references.

```suggestion
  /// The vertex format is `[x, y, z, u, v, w]`.
```